stream/redis.go

```diff
@@ -99,6 +99,8 @@ func (r RedisStream) Sub(ctx context.Context, stream string, id string, handleMe
 						}
 						continue
 					}
+
+					id = msg.ID
```

Comment:
This fixes a bug where the primary could accidentally skip an event on the redis stream